contrib/epee/include/storages/portable_storage_val_converters.h

```diff
@@ -146,7 +146,7 @@ POP_WARNINGS
         if(std::all_of(from.begin(), from.end(), ::isdigit))
           to = boost::lexical_cast<uint64_t>(from);
         // MyMonero ISO 8061 timestamp (2017-05-06T16:27:06Z)
-        else if (std::regex_match (from, std::regex("\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\dZ")))
+        else if (boost::regex_match (from, boost::regex("\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\dZ")))
```

Comment:
Why was this changed to `boost::regex`?

Reply:
All other code uses boost::regex, so it avoids including both.